graphs/hld/hld.cpp

```diff
@@ -0,0 +1,52 @@
+// define NEUT and oper
+int sz[N], dep[N], fat[N], pos[N], head[N];
+int cur_pos;
+
+void dfs_size(int from){
+  sz[from] = 1;
+  for(int to: adj[from]){
```

Comment:
The code must follow our convention in [here](https://google.github.io/styleguide/cppguide.html).
`for (int to : adj[from]) {`